src/main/java/com/hillel/items_exchange/exception/ElementsNumberExceedException.java

```diff
@@ -0,0 +1,11 @@
+package com.hillel.items_exchange.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_ACCEPTABLE)
+public class ElementsNumberExceedException extends Exception {
```

Comment:
We allready have AntityAmountException for this case (added in CGE-118). Would you like to use it or if you don't like name - just rename it, because they are identical

Reply:
I'll rename it. Thanks.